src/08-accessing.js

```diff
@@ -171,7 +193,22 @@ function getFullAddress(person) {}
       numbers: [7185550921, 7185558611],
     };
  */
-function getFlatObject(person) {}
+function getFlatObject(person) {
+  let newPerson = {
+    
+      first: `${person.names.first}`,
+      middle: `${person.names.middle}`,
+      last: `${person.names.last}`,
+      street: `${person.address.street}`,
+      unit: `${person.address.unit}`,
+      city: `${person.address.city}`,
+      state: `${person.address.state}`,
+      zip: `${person.address.zip}`,
+      numbers: person.numbers
+  }
+   return newPerson
+  }
```

Comment:
while this is not exactly hardcoded, there is a more dynamic way to do it! How can you use a loop to get the flat obj?